scrapers/R18.dev.yml

```diff
@@ -0,0 +1,428 @@
+name: R18.dev (JAV)
+
+sceneByFragment:
+  action: scrapeJson
+  queryURL: https://r18.dev/videos/vod/movies/detail/-/dvd_id={filename}/json
+  queryURLReplace:
+    filename:
+    # gets just the JAV ID out of the filename. This also removes the file extension which is pretty nice.
+    # You can have your filename be something like "Something Something ABC123 Something Something.mp4" and it will scrape as ABC123.
+      - regex: (.*[^a-zA-Z0-9])*([a-zA-Z-]+\d+)(.+)
+        with: $2
+  scraper: sceneSearchIndirect
+  
+sceneByURL:
+  - action: scrapeJson
+    url:
+      - r18.dev/videos/vod/movies/detail/-/i
+    scraper: sceneScraper
+    queryURL: "https://r18.dev/videos/vod/movies/detail/-/combined={url}/json"
+    queryURLReplace:
+      url:
+        - regex: '.+/id=(.+)/?$'
+          with: "$1"
+
+sceneByName:
+  action: scrapeJson
+  scraper: sceneSearch
+  queryURL: "https://r18.dev/videos/vod/movies/detail/-/dvd_id={}/json"
+
+sceneByQueryFragment:
+  action: scrapeJson
+  queryURL: "{url}"
+  scraper: sceneScraper
+  
+movieByURL:
+  - action: scrapeJson
+    url:
+      - r18.dev/videos/vod/movies/detail/-/i
+    queryURL: "https://r18.dev/videos/vod/movies/detail/-/combined={url}/json"
+    queryURLReplace:
+      url:
+        - regex: '.+/id=(.+)/?$'
+          with: "$1"      
+    scraper: movieScraper  
+
+jsonScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: dvd_id
+      Date:
+        selector: release_date
+      Performers:
+        Name: 
+          selector: actresses.#.name_romaji
+        Aliases:
+          selector: actresses.#.name_kanji
+      Image:
+        selector: jacket_full_url
+      Director:
+        selector: directors.#.name_romaji
+      Details:
+        selector: title_en
+        # All this uncensoring stuff happens automatically on the HTML version of the page due to some JS on the frontend there.
+        # I put in a request to the site dev to add this to the backend json api as well so we don't have to do it everytime ourselves plus keep a ever growing list of this stuff
```

Comment:
Sounds good. The issue was present in javlibrary and the original r18 site so we had to do that as well in the javlibrary_python scraper (you can have a look there for more word replacements if you want). I will merge the following as they are for now